src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  font-size: 12px;
+  font-family: Roboto, Helvetica, sans-serif;
```

Comment:
move font to `html` tag

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/201181669-0b825da9-c9cf-4786-9f17-e39687fab307.png)